immutable.js

```diff
@@ -35,6 +51,12 @@ console.log(combinedSupers);
   Create a new object `studentAverage` using Object.assign() from the given original object.
   The new object should filter out any scores that are < 34 Array.filter should be used here.
 */
-
+let studentScores = {
+  name: 'Igor',
+  age: 23,
+  scores: [33, 11, 66, 88, 99, 100],
+}
+let studentAverage = Object.assign({}, studentScores)
+studentAverage.scores = studentScores.scores.filter((score) => { return score >= 34 })
```

Comment:
If immediately returning out of an error function just remove the brackets and skip out on the return
`studentScores.scores.filter(score=> score >= 34 )`